lighthouse-extension/app/src/lighthouse-ext-background.js

```diff
@@ -231,8 +231,8 @@ function isRunning() {
   return lighthouseIsRunning;
 }
 
-// Run when in extension context, but not in devtools.
-if ('chrome' in window && chrome.runtime) {
+// Run when in extension context, but not in devtools or unit tests.
+if (typeof window !== 'undefined' && 'chrome' in window && chrome.runtime) {
```

Comment:
Before this PR, `lighthouse-ext-background.js` was pulling in `assetSaver`, which was pulling in `WebInspector`, which was setting `global.window = global`, so this file was fine to pull into node unit tests.

With that dependency path removed, this protects us from erroring on the `window` access (in the actual extension the behavior is unchanged since `window` is already defined :)